src/twisted/names/authority.py

```diff
@@ -34,28 +44,14 @@ def getSerial(filename = '/tmp/twisted-names.serial'):
         os.umask(o)
 
     with open(filename, 'r') as serialFile:
-        lastSerial, ID = serialFile.readline().split()
+        lastSerial, id = serialFile.readline().split()
 
-    ID = (lastSerial == serial) and (int(ID) + 1) or 0
+    id = (lastSerial == serial) and (int(id) + 1) or 0
 
     with open(filename, 'w') as serialFile:
-        serialFile.write('%s %d' % (serial, ID))
-
-#class LookupCacherMixin(object):
-#    _cache = None
-#
-#    def _lookup(self, name, cls, type, timeout = 10):
-#        if not self._cache:
-#            self._cache = {}
-#            self._meth = super(LookupCacherMixin, self)._lookup
-#
-#        if self._cache.has_key((name, cls, type)):
-#            return self._cache[(name, cls, type)]
-#        else:
-#            r = self._meth(name, cls, type, timeout)
-#            self._cache[(name, cls, type)] = r
-#            return r
-    serial = serial + ('%02d' % (ID,))
+        serialFile.write('%s %d' % (serial, id))
```

Comment:
can we use FIlePath here? I think that it should take care of handling the path encoding issues.